### PR TITLE
Indicator

### PR DESCRIPTION
**Pull request by ochanje210**

### CODE REVIEW

File: shared/containers/cardLayout.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import Card from '../components/Card';
 import { getDetailImage } from '../actions/listActions';
 import { setCurrentScene } from '../actions/fluxActions';
-import { saveEvent, deleteEvent} from '../actions/myPageActions'
+import { saveEvent, deleteEvent} from '../actions/myPageActions';
```
Comment (KyuaKwon) [inline shared/containers/cardLayout.js]: ```{ saveEvent, deleteEvent' '}```
blank needs to be added after elements

Comment (ochanje210) [inline shared/containers/cardLayout.js]: followed!

File: shared/components/map.js [hunk 2]
```diff
@@ -141,18 +160,29 @@ export default class Map extends Component {
...
+      this.setState({userLocationEnabled: true});
     },
     (error) => {
       // todo: handle this error when gps is off
       console.log(error);
+      this.setState({userLocationEnabled: false});
     });
   }
 
+  handleLocationButton() {
```
Comment (ochanje210) [inline shared/components/map.js]: a callback function to the buttons or user actions is better to be named handle+whattodo. Also, there should be another stuffs to be added in this handlelocationbutton function, for example, change zoom level when it is clicked.

Comment (ochanje210) [inline shared/components/map.js]: I will add todo comment

Comment (ochanje210) [inline shared/components/map.js]: added

Comment (yuda1124) [inline shared/components/map.js]: good! And I think that function's name is not clear. How about getUserLocation or etc..

**Comment by yuda1124** [comment]
LGTM

File: shared/components/map.js (hunk 2 above)
Comment (ochanje210) [inline shared/components/map.js]: we have named all our button onPress callback function starting with 'handle'.

**Pull request merged**